src/lambda/commands/invokeLambda.ts

```diff
@@ -136,7 +136,7 @@ function createMessageReceivedFunc({
     ...restParams
 }: {
     // TODO: Consider passing lambdaClient: LambdaClient
-    fn: FunctionNodeBase // TODO: Replace w/ invokeParams: {functionArn: string} // or Lambda.Types.InvocationRequest
+    fn: LambdaFunctionNode // TODO: Replace w/ invokeParams: {functionArn: string} // or Lambda.Types.InvocationRequest
```

Comment:
I didn't even know these TODOs existed. Is this something we still want to consider?